llvm/include/llvm/DebugInfo/DWARF/DWARFDebugLine.h

```diff
@@ -224,6 +227,8 @@ class DWARFDebugLine {
       return SectionIndex == PC.SectionIndex &&
              (LowPC <= PC.Address && PC.Address < HighPC);
     }
+
+    void SetSequenceOffset(uint64_t Offset) { StmtSeqOffset = Offset; }
```

Comment:
This is a struct, the members are all public - no need for a setter like this, just access/set the member directly?